psocake/ImagePanel.py

```diff
@@ -304,9 +307,12 @@ def updatePolarizationFactor(self): # FIXME: change to vertical polarization aft
             if self.parent.args.v >= 1: print "Done updatePolarizationFactor"
 
     def updateDetectorCentre(self):
-        self.parent.cy, self.parent.cx = self.parent.det.point_indexes(self.parent.evt, pxy_um=(0, 0), pix_scale_size_um=None,
-                                                       xy0_off_pix=None,
+        try:
+            self.parent.cy, self.parent.cx = self.parent.det.point_indexes(self.parent.evt, pxy_um=(0, 0), pix_scale_size_um=None,
+                                                       xy0_off_pix=None, fract=True,
```

Comment:
Hi @chuckie82 I just realized that one of the `fract=True` arguments need to be removed here.

```bash
Traceback (most recent call last):
  File "/reg/neh/home/fpoitevi/Toolkit/psocake/app/psocake", line 3, in <module>
    from psocake import gui
  File "/reg/neh/home/fpoitevi/Toolkit/psocake/psocake/gui.py", line 16, in <module>
    import MousePanel, ImagePanel, ImageStackPanel
  File "/reg/neh/home/fpoitevi/Toolkit/psocake/psocake/ImagePanel.py", line 313
    cframe=gu.CFRAME_PSANA, fract=True)
SyntaxError: keyword argument repeated
```

Reply:
thanks. if I'm not mistaken I've already fixed this in the latest psocake

Reply:
oh my bad! I checked and indeed you did (I thought I pulled it but did not)

Thanks!